rpm/SPECS/openresty-openssl.spec

```diff
@@ -35,26 +39,22 @@ Provides C header and static library for OpenResty's OpenSSL library.
 
 %prep
 %setup -q -n openssl-%{version}
-
 %patch0 -p1
 
 
 %build
-./config --prefix=%{openssl_prefix} \
-    no-threads \
-    shared -g
-
+./config --openssldir=%{_prefix} no-threads shared zlib -g \
```

Comment:
@agentzh Looking at https://wiki.openssl.org/index.php/Compilation_and_Installation#PREFIX_and_OPENSSLDIR I interpret that --openssldir is preferable.